gateways/authorizenet/request_builders.go

```diff
@@ -37,7 +38,25 @@ func buildAuthRequest(merchantName string, transactionKey string, authRequest *s
 
 	authorizeRequest := CreateTransactionRequest{
 		MerchantAuthentication: authentication(merchantName, transactionKey),
-		TransactionRequest: TransactionRequest{
+	}
+
+	var transactionRequest TransactionRequest
+	if authRequest.Options[sleet.GooglePayTokenOption] != nil {
+		// Google Pay request
+		googlePayToken := authRequest.Options[sleet.GooglePayTokenOption].(string)
+		encodedGooglePayToken := base64.StdEncoding.EncodeToString([]byte(googlePayToken))
+		transactionRequest = TransactionRequest{
+			TransactionType: TransactionTypeAuthCapture,
```

Comment:
Just curious, does this mean auto capture? or this is just specific for GooglePay?

Reply:
Specific to Google Pay. https://developer.authorize.net/api/reference/index.html#mobile-in-app-transactions-create-a-google-pay-transaction.

Reply:
But based on https://developer.authorize.net/api/reference/features/payment_transactions.html, I think this seems to be Auth + Capture. Would https://developer.authorize.net/api/reference/index.html#mo just list one example? Could you check on this?